ui/job-view/JobView.jsx

```diff
@@ -232,74 +244,87 @@ class JobView extends React.Component {
+                    getAllShownJobs={this.getAllShownJobs}
+                    duplicateJobsVisible={duplicateJobsVisible}
+                    groupCountsExpanded={groupCountsExpanded}
                     $injector={$injector}
                   />
-                </SplitPane>
-                <NotificationList />
-              </KeyboardShortcuts>
-            </SelectedJob>
-          </PinnedJobs>
-        </Pushes>
-      </Notifications>
+                  <SplitPane
+                    split="horizontal"
+                    size={`${pushListPct}%`}
+                    onChange={size => this.handleSplitChange(size)}
+                  >
+                    <div className="d-flex flex-column w-100">
+                      {(isFieldFilterVisible || !!filterBarFilters.length) && <ActiveFilters
+                        $injector={$injector}
+                        classificationTypes={classificationTypes}
+                        filterModel={filterModel}
+                        filterBarFilters={filterBarFilters}
+                        isFieldFilterVisible={isFieldFilterVisible}
+                        toggleFieldFilterVisible={this.toggleFieldFilterVisible}
+                      />}
+                      {serverChangedDelayed && <UpdateAvailable
+                        updateButtonClick={this.updateButtonClick}
+                      />}
+                      <div id="th-global-content" className="th-global-content" data-job-clear-on-click>
+                        <span className="th-view-content" tabIndex={-1}>
+                          <PushList
+                            user={user}
+                            repoName={repoName}
+                            revision={revision}
+                            currentRepo={currentRepo}
+                            filterModel={filterModel}
+                            duplicateJobsVisible={duplicateJobsVisible}
+                            groupCountsExpanded={groupCountsExpanded}
+                          />
+                        </span>
+                      </div>
+                    </div>
+                    <DetailsPanel
+                      resizedHeight={detailsHeight}
+                      currentRepo={currentRepo}
+                      repoName={repoName}
+                      user={user}
+                      classificationTypes={classificationTypes}
+                      classificationMap={classificationMap}
+                      $injector={$injector}
+                    />
+                  </SplitPane>
+                  <NotificationList />
+                  {showShortCuts && <div
+                    id="onscreen-overlay"
+                    onClick={() => this.showOnScreenShortcuts(false)}
+                  >
+                    <div id="onscreen-shortcuts">
+                      <div className="col-8">
+                        <ShortcutTable id="th-shortcut-table" />
```

Comment:
Both this and the user guide usage set the same `id` -- could it be moved down into the `ShortcutTable` component?